hw/vendor/esl_epfl_x_heep/.gitignore

```diff
@@ -32,3 +42,7 @@ sw/linker/link_flash_load.ld
 sw/device/lib/drivers/power_manager/power_manager_regs.h
 sw/device/lib/drivers/power_manager/power_manager.h
 sw/device/lib/drivers/pad_control/pad_control_regs.h
+sw/device/lib/drivers/**/*_structs.h
```

Comment:
Some of the ignore file paths are not existing. If the x-heep gitignore file is vendored we can remove all these paths from here